src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const types = [];
```

Comment:
types of what? let's give it another name

Reply:
change types to history. Done.